docs/jwst/associations/jwst_conventions.rst

```diff
@@ -60,4 +60,4 @@ The current association types are:
   * `tso3`: Intended for `calwebb_tso3` processing
   * `image2`: Intended for `calwebb_image2` processing
   * `spec2`: Intended for `calwebb_spec2` processing
-  * `spec2-nrslamp`: Intended for `calwebb_spec2_nrslamp` processing
+  * `nrslamp-spec2`: Intended for `calwebb_nrslamp_spec2` processing
```

Comment:
A really minor nit, but do the other asn type names also use hyphens, as opposed to underscores? The config file names use underscores.

Reply:
I believe the file naming conventions for association names require that this be a hyphen since this all goes in the `type` field of the name:

```
jw<ProgramID>-<AC_ID>_<GT-DATETIME>_<type>_<mmm>_asn.json
```

Reply:
Ah, good point. 👍